integration_tests/src/util/workflows.rs

```diff
@@ -879,7 +961,37 @@ pub fn verifreg_remove_expired_allocations(
     ids: Vec<AllocationID>,
     datacap_refund: u64,
 ) {
+    let v_st: VerifregState = get_state(v, &VERIFIED_REGISTRY_ACTOR_ADDR).unwrap();
+    let store = DynBlockstore::wrap(v.blockstore());
+    let mut allocs = v_st.load_allocs(&store).unwrap();
+    let mut expected_events: Vec<EmittedEvent> = ids
+        .iter()
+        .map(|id| {
+            let alloc = allocs.get(client.id().unwrap(), *id).unwrap().unwrap();
+            build_verifreg_event("allocation-removed", *id, client.id().unwrap(), alloc.provider)
+        })
+        .collect();
+
+    if ids.is_empty() {
+        let expired =
+            expiration::find_expired(&mut allocs, client.id().unwrap(), v.epoch()).unwrap();
```

Comment:
Suggestion: as an alternative, inspect whether the caller/s of this method already know which allocs to expect expired. If it would be easy for them to pass in, add a parameter for that rather than inspecting the state and duplicating/calling the same logic here. The caller clearly knows something about the datacap refund to expect

Reply:
Done.